yaksh/templatetags/custom_filters.py

```diff
@@ -62,3 +63,21 @@ def module_completion_percent(course, module, user):
 @register.simple_tag
 def course_completion_percent(course, user):
     return course.percent_completed(user)
+
+
+@register.simple_tag
+def get_ordered_testcases(question, answerpaper):
+	return question.get_ordered_test_cases(answerpaper)
+
+@register.simple_tag
+def get_arrange_user_answer(ans, question):
+    if type(ans) == bytes:
+        ans = ans.decode("utf-8")
+    else:
+        ans = str(ans)
+    ans_list = literal_eval(ans)
+    testcase_list = []
```

Comment:
The variable in question contains a list of testcases, hence the name.

Reply:
You can write ```testcases = []```
Because reading the code it is understood that testcases is a ```list``` and no need to mention in the variable name!

Reply:
I agree, no need to mention the type in the variable name.

Reply:
@maheshgudi 
I think you missed to resolve this comment.